sdk/lib/rtl/heap.c

```diff
@@ -353,21 +353,23 @@ RtlpInsertFreeBlockHelper(PHEAP Heap,
     Heap->FreeHints[HintIndex] = &FreeEntry->FreeList;
 }
 
-VOID NTAPI
+LONG NTAPI
 RtlpInsertFreeBlock(PHEAP Heap,
                     PHEAP_FREE_ENTRY FreeEntry,
                     SIZE_T BlockSize)
 {
     USHORT Size, PreviousSize;
     UCHAR SegmentOffset, Flags;
     PHEAP_SEGMENT Segment;
+    LONG FreeSize;
```

Comment:
LONG ? Shouldn't this be ULONG instead?

Reply:
This is OK. See my reasoning below. Thanks.

Reply:
Now Removed. Thanks.